grammars/blade.cson

```diff
@@ -11,21 +11,25 @@
     'patterns': [
       {
         'begin': '{{--'
-        'captures':
+        'beginCaptures':
           '0':
-            'name': 'punctuation.definition.comment.blade'
+            'name': 'punctuation.definition.comment.begin.blade'
         'end': '--}}'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.definition.comment.end.blade'
         'name': 'comment.block.meta.embedded.blade'
       }
       {
         'begin': '(?<!@){{{'
-        'captures':
+        'beginCaptures':
           '0':
-            'name': 'support.punctuation.blade'
-          '1':
-            'name': 'source.php'
```

Comment:
Leave these kind captures in, this is a hack required for text editors for properly detect inner scope when the contents of the block is 0 width, like `{{|}}` where `|` is cursor.